src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PackageRestore/DropConfiguredProjectVersionDataSource2.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System;
+using System.Threading.Tasks.Dataflow;
+using Microsoft.VisualStudio.ProjectSystem.VS.PackageRestore;
+
+namespace Microsoft.VisualStudio.ProjectSystem
+{
+    /// <summary>
+    ///     A <see cref="IProjectValueDataSource{T}"/> that drops <see cref="ProjectDataSources.ConfiguredProjectIdentity"/> and
+    ///     <see cref="ProjectDataSources.ConfiguredProjectVersion"/> versions from each value of the original
+    ///     <see cref="IProjectValueDataSource{T}"/>.
+    /// </summary>
+    internal sealed class DropConfiguredProjectVersionDataSource2<T> : ChainedProjectValueDataSourceBase<T>
```

Comment:
We don't usually put a `2` suffix on classes (we usually version interfaces). What is the reason for this? Can we have a more descriptive name instead?